### DISCUSSION TITLE
[@types/node] overwrites the DOM global Blob with incorrect signatures

### DISCUSSION THREAD

**Discussion by glenjamin** [DefinitelyTyped/DefinitelyTyped#58079] [Issues with a @types package]
In `stream/consumers.d.ts`:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/75f6dff53de980e934bb1117d431f3a8259ae771/types/node/stream/consumers.d.ts#L1-L13

This overwrites the one from `lib.dom.d.ts`

https://github.com/microsoft/TypeScript/blob/b53073bf66571b4c99f0a93b8f76fb06b36d8991/lib/lib.dom.d.ts#L2417-L2425

the global `Blob`'s `stream()` method is defined as returning a `NodeJS.ReadableStream`, but as far as I can tell NodeJS doesn't have a global Blob class - so I'm not sure why this is done?

I'm running into this because I have a project with a mix of NodeJS and browser files, I did try and see if I could workaround this by making sure that `@types/node` only applies to the relevant files, but I couldn't figure that out.

[a comment by an automated account is not shown]

**Comment by thw0rted** [comment]
I submitted a fix for this in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59905, I'm resolving some outstanding issues but if it gets accepted it should resolve your problem.
